client/jetpack-cloud/sections/partner-portal/issue-license-v2/licenses-form/hooks/use-product-and-plans.tsx

```diff
@@ -110,7 +110,7 @@ const getDisplayableProducts = ( filteredProductsAndBundles: APIProductFamilyPro
 		const product_a = Array.isArray( a ) ? a[ 0 ].name : a.name;
 		const product_b = Array.isArray( b ) ? b[ 0 ].name : b.name;
 		return product_a.localeCompare( product_b );
-	} );
+	} ) as APIProductFamilyProduct[];
```

Comment:
Because the `type` is set as `APIProductFamilyProduct` or `APIProductFamilyProduct []`, it causes a lot of type issues wherever this is used.